### DISCUSSION TITLE
Has anyone ever successfully export the maps to Blender?

### DISCUSSION THREAD

**Discussion by Maciox55** [etlegacy/etlegacy#2177] [General]
Im on a quest... I'd like to import OG ET maps into Blender so I can mess around with some rendering. I did some digging, found some tools but none seem to work correctly. Im able to import items, etc. but not the actual maps themselves. Anyone have any pointers how to do this?

**Comment by WuTangH** [comment]
Hi, 
there is no direct MAP importer to Blender as far as I know. The best way would be to use NetRadiants OBJ exporter and import the .obj model to Blender then. But make sure to use NetRadiant. GtkRadiant provides an exporter too, but it needs lot of cleanup in Blender.
https://github.com/Garux/netradiant-custom/releases

There are also some BSP importers for Blender, but I dont think they work with ETs .bsp:  
https://github.com/SomaZ/Blender_BSP_Importer/releases
https://github.com/andyp123/blender_io_mesh_bsp

**Reply by wpn1966** [reply]
Speaking of maps, I am new to GitHub and new to Enemy Territory Maps. I have recently started playing around with NetRadiant 1.5.0n-git-d9338309e (Jun 28 2022), and I have created my first map. I found the experience was fun, and I think I want to continue doing it as a hobby in my spare time. Do you have any helpful thoughts or suggestions for a newbie?  Also, are there any map repositories, which include the actual .map file and not just the .bsp for learning purposes? Thanks, in advance.

**Reply by WuTangH** [reply]
Hi! 
My suggestion would probably be to read about map optimization, especially about difference between detail & structural brushes. Check maybe this one, but it might look confusing at start  - http://www.robotrenegade.com/articles/id-tech-3-optimization/geometry-optimizations.html

Here are some map sources that you can learn from:
Original maps: https://www.wolffiles.de/index.php?filebase&fid=1569
Capuzzo Airport: https://www.wolffiles.de/index.php?filebase&fid=4833
MLB maps: https://www.wolffiles.de/index.php?filebase&fid=4304
Baserace: https://www.wolffiles.de/index.php?filebase&fid=1374
Streets of Italy: https://www.wolffiles.de/index.php?filebase&fid=1518

...and the ultimate resource pack by Teuthis: https://www.wolffiles.de/index.php?filebase&fid=5075
...and a big Idtech3 mapping database: https://trello.com/b/zJp4pE3m/id-tech-3-mapping

And some more links:
http://www.pythononline.co.uk/et/tutorial.htm
http://games.chruker.dk/enemy_territory/entities.php
http://q3map2.robotrenegade.com/docs/shader_manual/contents.html
https://wiki.splashdamage.com/index.php/Wolfenstein:_Enemy_Territory

Hope it helps!

**Reply by wpn1966** [reply]
Oh, this is AWESOME! Thank you, so much for those links!  Today I added a neutral command post to my little test map, after following the tutorial at http://www.pythononline.co.uk/et/tutorial20.htm I now understand a little about the speaker definitions in the .sps file. There is a great joy in learning something new! :)
